### PR TITLE
Add pattern issue template

### PR DESCRIPTION
**Pull request by substrae**
Adds an issue template for patterns. To be used when creating pattern discussions.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/⌨-accessibility-issue.md [hunk 1]
```diff
@@ -2,8 +2,7 @@
 name: "⌨ Accessibility Issue"
 about: An accessibility related issue template.
 title: "⌨ Accessibility Issue"
-labels: 
-  - accessibility
+labels: accessibility, component, epic
```
Comment (esizer) [inline .github/ISSUE_TEMPLATE/⌨-accessibility-issue.md:5]: What was the reasoning behind using the `epic` label here?

Comment (substrae) [reply .github/ISSUE_TEMPLATE/⌨-accessibility-issue.md:5]: Actually, it shouldn't be 🤔 Something wonky happened with Github's templating UI in both these PRs (deleting a previous template in the other PR, adding labels from one template to another in this PR), so maybe we retain template PRs for instances like this.

Comment (esizer) [reply .github/ISSUE_TEMPLATE/⌨-accessibility-issue.md]: 🤷‍♀️ That is odd, I have never used this feature tbh so I have no clue 😛

**Review by esizer — APPROVED** [review]
Looking good! 👍

**Pull request merged**